codeguard/cg_feedback_collection
# Style updates

## How to test

1. Navigate through the various customer pages and admin pages.
2. Make sure that no major layout issues crop up.
